src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -19,7 +19,25 @@
     <DeployExtension>false</DeployExtension>
     <UseCodebase>true</UseCodebase>
     <CopyNuGetImplementations>false</CopyNuGetImplementations>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
   </PropertyGroup>
+  <!-- We need to mark our NuGet.VisualStudio package reference as EmbedInteropTypes.  However,
+       as this package comes from project.json there is no way to do that in the json file.
+       To deal with that, we write this target which finds that added package, removes it from
+       the references list, then adds it back in again with the 'EmbedInteropTypes' flag properly
+       set. -->
+  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolveNuGetPackageAssets">
```

Comment:
Ah, I take back my previous comment about getting this out of the project.json.  I hadn't seen this gem @jasonmalinowski came up with...